cips/cip-22.md

```diff
@@ -1,9 +1,9 @@
 | cip | 22 |
 | - | - |
-| title | Removing the blobStartIndex |
+| title | Removing the IndexWrapper |
```

Comment:
[nit] can you update the title here too: https://github.com/celestiaorg/CIPs/blob/0301c23f801a70c72df532798a22f32b4fa70780/cips/README.md?plain=1#L47